### PR TITLE
Add function to enable huge pages

### PR DESCRIPTION
**Pull request by petr-balogh**
Signed-off-by: Petr Balogh <pbalogh@redhat.com>

### CODE REVIEW

File: ocs_ci/ocs/constants.py [hunk 1]
```diff
@@ -68,9 +68,13 @@
 ROOK_CSI_CEPHFS_DIR = os.path.join(ROOK_EXAMPLES_DIR, "csi", "cephfs")
 CLEANUP_YAML = "cleanup.yaml.j2"
 MANIFESTS_DIR = "manifests"
+
+
```
Comment (vavuthu) [inline ocs_ci/ocs/constants.py:72]: remove blank line here

File: docs/usecases/enable_huge_pages.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
+# ENABLE HUGE PAGES
+
+If you need to enable Huge Pages please use this config file:
+
+`ocs_ci/templates/ocp-deployment/huge_pages.yaml`
+
+If you pass this file via --ocsci-conf, it will apply this file:
+
+`ocs_ci/templates/ocp-deployment/huge_pages.yaml`
```
Comment (vavuthu) [inline docs/usecases/enable_huge_pages.md:9]: we need to enable huge pages using conf/ocsci/enable_huge_pages.yaml

**Comment by vavuthu** [comment]
> PR validation
> 
> Cluster Name:
> Cluster Configuration: ocs_ci/templates/ocp-deployment/huge_pages.yaml
> PR Test Suite: deployment
> PR Test Path: tests/
> Additional Test Params:
> OCP VERSION: 4.7
> OCS VERSION: 4.7
> 
> [Job FAILED](https://ocs4-jenkins-csb-ocsqe.apps.ocp4.prod.psi.redhat.com/job/qe-trigger-test-pr/949//testReport) (installation failed, tests not executed).

looks like you have passed wrong conf file

File: ocs_ci/ocs/constants.py (hunk 1 above)
Comment (petr-balogh) [reply ocs_ci/ocs/constants.py]: Sometimes there were two empty lines between sections, sometime one, I deleted it on every place were we had 2 and we should now follow only one. Thanks

File: docs/usecases/enable_huge_pages.md (hunk 2 above)
Comment (petr-balogh) [reply docs/usecases/enable_huge_pages.md]: Done, yep, I copied wrong file  :( the same in verification

**Comment by petr-balogh** [comment]
Verified here:
https://ocs4-jenkins-csb-ocsqe.apps.ocp4.prod.psi.redhat.com/job/qe-deploy-ocs-cluster/4056/console

**Review by vavuthu — APPROVED** [review]

**Review by dahorak — APPROVED** [review]

**Pull request merged**